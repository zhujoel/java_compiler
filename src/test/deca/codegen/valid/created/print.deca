{
    print("Hello");
    print(5);
    int a = 4;
    print(a);
    print(2.0);
    float b = 1.2;
    print(b);
    print(true);
    print(false);
    bool c = true;
    print(c);
}
