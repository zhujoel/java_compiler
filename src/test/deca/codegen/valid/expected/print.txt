Hello542.00000e01.20000e0
